setup.py

```diff
@@ -48,6 +48,7 @@
     "google-cloud-spanner==3.1.0",
     "setuptools>=34.0.0",
     "jellyfish==0.8.2",
+    "tabulate==0.8.9",
```

Comment:
Where do you use tabulate? It seems like you only use the built-in pandas.DataFrame methods

Reply:
table printing internally used tabulate

`print(result_df.to_markdown(tablefmt="fancy_grid"))`